src/debugger/debugger.c

```diff
@@ -1006,6 +1021,8 @@ PHP_FUNCTION(xdebug_connect_to_client)
 
 	XG_DBG(context).do_connect_to_client = 1;
 
+	xdebug_enable_debugger_and_rebuild_stack_if_disabled();
```

Comment:
If we have been told to connect to the client using the xdebug_connect_to_client() function, we re-enable the debugger and rebuild the stack if needed